sympy/series/tests/test_order.py

```diff
@@ -405,3 +405,7 @@ def test_order_subs_limits():
 
     assert Order(x**2).subs(x, y - 1) == Order((y - 1)**2, (y, 1))
     assert Order(10*x**2, (x, 2)).subs(x, y - 1) == Order(1, (y, 3))
+
+
+def test_issue_9192():
+    assert O(1)*O(1) == O(1)
```

Comment:
Does this also fix the other issue? If so, you should add a test for it as well:

```
>>> O(1) ** O(1) # should be O(1)
O(1)**O(1) 
```